backend/src/main/resources/application.properties

```diff
@@ -91,3 +91,8 @@ quarkus.rest-client.sources.url=${clowder.endpoints.sources-api-svc:http://local
 # OpenTelemetry -- see also jdbc driver above.
 quarkus.opentelemetry.enabled=false
 quarkus.opentelemetry.tracer.exporter.otlp.endpoint=http://localhost:4317
+
+# Sources' development PSK value. Specified here to avoid Quarkus from complaining that the configuration parameter
+# is missing. In the case that you are using a real Sources application to test the integration, you will need to
+# set this PSK in the SOURCES_PSKS environment variable on that end.
+sources.psk=development-value-123
```

Comment:
When does Quarkus complain exactly? Is it during the tests execution, in dev mode, in prod mode?

Reply:
When running with `quarkus:dev` Quarkus doesn't start unless this value is specified, since I didn't provide a default value for the `@ConfigProperty`. The same happens when running tests, the dependency injection starts going a bit crazy, and it happens when I comment the property and recompile the project.